public/models.json

```diff
@@ -0,0 +1,1231 @@
+        },
+        "tr": {
+            "languages": ["tr"],
+            "title": "Türkçe"
+        },
+        "tt": {
+            "languages": ["tt"],
+            "title": "татарча/tatarça"
+        },
+        "uk": {
+            "languages": ["uk"],
+            "title": "українська"
+        },
+        "uz": {
+            "languages": ["uz"],
+            "title": "oʻzbekcha/ўзбекча"
+        },
+        "uz-Cyrl": {
+            "languages": ["uz-cyrl"],
+            "title": "ўзбекча"
+        },
+        "vi": {
+            "languages": ["vi"],
+            "title": "Tiếng Việt"
+        },
+        "yi": {
+            "languages": ["yi"],
+            "title": "ייִדיש"
+        },
+        "yo": {
+            "languages": ["yo"],
+            "title": "Yorùbá"
+        },
+        "zh": {
+            "languages": ["zh"],
+            "title": "中文"
+        },
+        "zu": {
+            "languages": ["zu"],
+            "title": "isiZulu"
+        }
+    },
+    "tesseract": {
+        "afr": {
+            "languages": ["af"],
+            "title": "Afrikaans"
+        },
+        "amh": {
+            "languages": ["am"],
+            "title": "አማርኛ"
+        },
+        "ara": {
+            "languages": ["ar"],
+            "title": "العربية"
+        },
+        "asm": {
+            "languages": ["as"],
+            "title": "অসমীয়া"
+        },
+        "aze": {
+            "languages": ["az"],
+            "title": "azərbaycanca"
+        },
+        "az-cyrl": {
+            "languages": ["aze","cyrl"],
+            "title": "Azerbaijani (Cyrillic)"
+        },
+        "bel": {
+            "languages": ["be"],
+            "title": "беларуская"
+        },
+        "bul": {
+            "languages": ["bg"],
+            "title": "български"
+        },
+        "ben": {
+            "languages": ["bn"],
+            "title": "বাংলা"
+        },
+        "bod": {
+            "languages": ["bo"],
+            "title": "བོད་ཡིག"
+        },
+        "bre": {
+            "languages": ["br"],
+            "title": "brezhoneg"
+        },
+        "bos": {
+            "languages": ["bs"],
+            "title": "bosanski"
+        },
+        "cat": {
+            "languages": ["ca"],
+            "title": "català"
+        },
+        "ceb": {
+            "languages": ["ceb"],
+            "title": "Cebuano"
+        },
+        "chr": {
+            "languages": ["chr"],
+            "title": "ᏣᎳᎩ"
+        },
+        "ces": {
+            "languages": ["cs"],
+            "title": "čeština"
+        },
+        "cos": {
+            "languages": ["cos"],
+            "title": "Corsican"
+        },
+        "cym": {
+            "languages": ["cy"],
+            "title": "Cymraeg"
+        },
+        "dan": {
+            "languages": ["da"],
+            "title": "dansk"
+        },
+        "deu": {
+            "languages": ["de"],
+            "title": "Deutsch"
+        },
+        "frk": {
```

Comment:
I renamed this Tesseract model from frk to deu_latf.

Reply:
I've added a separate duplicate entry for deu_latf, and skipped it in the tests for now.